jobs/scanning/covscan-images/Jenkinsfile

```diff
@@ -68,6 +68,12 @@ timeout(activity: true, time: 1, unit: 'DAYS') {  // if there is no log activity
                             defaultValue: false,
                             description: "Force rebuild of the covscan repositories."
                         ),
+                        [
+                            name: 'HTTPS_PROXY',
```

Comment:
It will be a burden for user to remember which proxy is current and set this build parameter for every job run.
Suggest removing this parameter, but read the env var from the build env.

Reply:
Done